### PR TITLE
Fixes AB#412 disable menu for unverified user

### PR DESCRIPTION
**Pull request by Skafec**
Merged new master in branch

### CODE REVIEW

File: src/app/sidebar/sidebar.component.html [hunk 1]
```diff
@@ -10,19 +10,19 @@
             <i class="menu-icon fas fa-align-left"></i>
             <span>Offers</span>
         </li>
-        <li routerLink="my_portfolio" routerLinkActive="side-active">
+        <li routerLink="my_portfolio" routerLinkActive="side-active" [ngClass]="{'unverified-user-disabled': !hasVerifiedProfile}">
```
Comment (mj-ampnet) [inline src/app/sidebar/sidebar.component.html:13]: You can use the syntax `[class.unverified-user-disabled]="!hasVerifiedProfile"` which is shorter and looks much cleaner.

**Review by filipdujmusic — DISMISSED** [review]

File: src/app/sidebar/sidebar.component.ts [hunk 2]
```diff
@@ -56,6 +56,7 @@ export class SidebarComponent implements OnInit {
     fetchUserData() {
         this.userService.getOwnProfile().subscribe(res => {
             this.fullName = res['first_name'] + ' ' + res['last_name'];
+            this.hasVerifiedProfile = res['verified'];
```
Comment (mj-ampnet) [inline src/app/sidebar/sidebar.component.ts:59]: Has this been tested that this function is called _after_ the user finishes Identyum verification? Was this achieved by @mdrusko1 with the `refresh` functionality in other pull request maybe? (not sure, just asking)

File: src/app/sidebar/sidebar.component.css [hunk 3]
```diff
@@ -84,3 +84,8 @@ a {
     padding-left: 24px;
     font-size: 12px;
 }
+
+.unverified-user-disabled {
+    pointer-events: none;
+    color: #e6e6e6 !important;
+}
```
Comment (mj-ampnet) [inline src/app/sidebar/sidebar.component.css:91]: End of file, add one new line

File: src/app/sidebar/sidebar.component.ts (hunk 2 above)
Comment (mightymatth) [reply src/app/sidebar/sidebar.component.ts:59]: Yes, I'm concerned about that too. 

As the sidebar component stays the same during the entire application session, it should be notified about changing user data from outside. I propose creating a [Subject](https://www.learnrxjs.io/learn-rxjs/subjects/subject) inside user service which will act as a notifier for any other component that needs to change when the user data changes from outside. For example, when a user token needs to be changed, you will get new data and pass it to the subject. Now, inside the sidebar component, inject user service and subscribe to this subject which will tell the sidebar component when to refresh the user data.

Tell me if it's confusing. :)

**Review by mightymatth — CHANGES_REQUESTED** [review]

**Pull request closed without merging**